Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -281,7 +281,9 @@ private func topFilters(forUser user: User?) -> [DiscoveryParams] {
 
   if user != nil {
     filters.append(.defaults |> DiscoveryParams.lens.starred .~ true)
-    filters.append(.defaults |> DiscoveryParams.lens.recommended .~ true)
+    filters.append(.defaults 
+                   |> DiscoveryParams.lens.recommended .~ true 
+                   <> DiscoveryParams.lens.backed .~ false)
```

Comment:
nice! our general code style when using multiple lenses within `()` is to drop it to its own new section, i.e.
```swift
filters.append(
  .defaults
    |> DiscoveryParams.lens.recommended .~ true
    |> DiscoveryParams.lens.backed .~ false
)
```